cmd/topology.go

```diff
@@ -108,5 +114,16 @@ func runTopology(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	openCmd := exec.Command("open", topologyOutputFile)
```

Comment:
On my machine:
`Error: exec: "open": executable file not found in $PATH`
The .svg and .svg.dot files are stored on my disk as expected.

Reply:
Fixed, can you try once more?